### DISCUSSION TITLE
Thank You Corey!

### DISCUSSION THREAD

**Discussion by calebwebster** [coreybutler/nvm-windows#791] [General] [answered]
Hi Corey,

Computer science student from Australia here. I've recently gotten into Nodejs and being a Windows user, I discovered your nvm project. After using it, I was impressed at what a wonderful and simple piece of software it is! It is a joy to use and I think it's cool how you chose to use Go to make a better product than others.  Anyway, I just wanted to say: Thank you for making this piece of software. I hope to one day do something like what you are doing with nvm, something that improves people's lives (maybe I'll start by contributing to this project :)).

Cheers,
Caleb.

**Comment by coreybutler** [comment, marked as answer]
Thanks Caleb - this really means alot to me. I'm glad you've found value in the software. Welcome to the world of Node!

Contributions are always welcome. For you and anyone else reading, this project would benefit most from having folks who can answer user questions. I am trying to work on an NVM4W successor, called Runtime (rt). Between Runtime, NVM4W, & other OSS projects (plus a packed work schedule), there isn't much time to answer questions, do code reviews, and _test_ PRs.
